packages/playwright/src/worker/testInfo.ts

```diff
@@ -456,6 +459,27 @@ export class TestInfoImpl implements TestInfo {
     this._timeoutManager.setIgnoreTimeouts();
   }
 
+  async _didFinishTestFunction() {
+    if (this._pauseAtEnd() === 'notify' && !this._isFailure()) {
```

Comment:
Given `pauseAtEnd: notify, pauseOnError: off`, when a failure occurs, I don't think it pauses currently. Is that intended?